views/templates/panes/compiler.pug

```diff
@@ -55,6 +55,7 @@ mixin newPaneButton(classId, text, title, icon)
         +newPaneButton("add-compiler", "Clone Compiler", "Clone this compiler window (click or drag)", "far fa-clone")
         +newPaneButton("create-executor", "Executor From This", "Create executor from this compiler", "fas fa-microchip")
         +newPaneButton("view-optimization", "Optimization", "Show optimization output", "fas fa-weight")
+        +newPaneButton("view-stack-usage", "Stack Usage", "Show stack usage", "fas fa-microchip")
```

Comment:
Maybe one of these icons can be used? (Just to visually differentiate from the executor)
![image](https://github.com/compiler-explorer/compiler-explorer/assets/51220084/e4ddff84-c83f-4545-9941-8de83715f7e5)